src/App.tsx

```diff
@@ -1,28 +1,63 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodosFromServer } from './api';
 
 const App: React.FC = () => {
   const [
     selectedUserId,
-    // setSelectedUserId,
-  ] = useState(0);
+    setSelectedUserId,
+  ] = useState<null | number>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<null | string>(null);
+
+  const clearUser = useCallback(() => {
+    setSelectedUserId(null);
+  }, []);
+
+  useEffect(() => {
+    const receiveTodosPromise = getTodosFromServer();
+
+    receiveTodosPromise
+      .then((receiveTodos) => {
+        setTodos(receiveTodos);
+      })
+      .catch((fetchError) => {
```

Comment:
async await gives you a simpler way to work with asynchronous promise-based code